### DISCUSSION TITLE
Ctrl - Shift - Left Click: Move All types of same Currency from Inventory

### DISCUSSION THREAD

**Discussion by LordStorm7** [Lailloken/Exile-UI#442] [Q&A]
Does anyone know how to get this work again in POE? I downloaded this tool for the ACT walk-thru and it was Great for that, but now I can't 1 click transfer currency and I'm sure it has to do with this programs key-binds, but I can't find where or how to change it?
Any help would be greatly appreciated!

**Comment by Lailloken** [comment]
You got things mixed up:
- moving all currency is ctrl + right-click

- ctrl + shift + left-click is moving something into the currently-open tab while ignoring tab affinities
